### ISSUE TITLE
hash filenames with ipx

### ISSUE THREAD

**Issue by ghostlexly** [nuxt/image#383]
Hello,

IPX is generating paths like:
/_ipx/images/homepage/top-slider/logo-colipays-bordeaux-gris.png?fit=fill&s=200_50

But the image file name doesn't change. How can be add hash/chunk to the file name with ipx so the file will not be cached after a new deploy.

Example:
/_ipx/images/homepage/top-slider/logo-colipays-bordeaux-gris..19dm102lf.png?fit=fill&s=200_50

**Comment by kevinpainchaud** [comment]
It would be very interesting to be able to do this in order to cache images longer (#329).

**Comment by ghostlexly** [comment]
I have found a way to do it but it's really overkill...

**Comment by rwwagner90** [comment]
@ghostlexly how did you accomplish adding a hash?

**Comment by mikereinhold** [comment]
> I have found a way to do it but it's really overkill... For now, my issue is fixed but i'm open to a any more easy methods.

@ghostlexly can you share any details on the approach that you took?

**Comment by ojvribeiro-kore** [comment]
It's really frustrating when someone says they found a solution and just vanish without sharing it with others.

**Comment by Red-Asuka** [comment]
In the Nuxt2 version of this plugin, there is a `staticFilename` configuration that supports hash. However, it seems that this configuration is no longer available in the new version.

**Comment by Hibrix-net** [comment]
Supporting hash is essential. Someone should take care of implementing it asap.

**Comment by Hibrix-net** [comment]
> Sorry for my late reply, I do not receive notifications on GitHub. If anyone comes across this and is looking for the solution, here it is:
> 
> ```js
> export default {
>   image: {
>       // Generate images to `/_nuxt/image/file.png`
>       staticFilename: '[publicPath]/images/[name]-[hash][ext]'
>   }
> }
> ```

Hi,
As @Red-Asuka already pointed above:
There's no such a property for Nuxt3: https://image.nuxt.com/get-started/configuration
`staticFilename` is only available in the previous versions of Nuxt Image, the ones compatible with Nuxt2: https://v0.image.nuxtjs.org/api/options#staticfilename

**Comment by ghostlexly** [comment]
> > Sorry for my late reply, I do not receive notifications on GitHub. If anyone comes across this and is looking for the solution, here it is:
> > ```js
> > export default {
> >   image: {
> >       // Generate images to `/_nuxt/image/file.png`
> >       staticFilename: '[publicPath]/images/[name]-[hash][ext]'
> >   }
> > }
> > ```
> 
> Hi, As @Red-Asuka already pointed above: There's no such a property for Nuxt3: https://image.nuxt.com/get-started/configuration `staticFilename` is only available in the previous versions of Nuxt Image, the ones compatible with Nuxt2: https://v0.image.nuxtjs.org/api/options#staticfilename

You're right, my problem was on Nuxt2. I do not know the solution for Nuxt3.

**Comment by zumm** [comment]
Any updates? This feature is really desired.
